cfgov/unprocessed/css/nav-secondary.less

```diff
@@ -132,123 +118,109 @@
           -----------------------
           .nav-secondary.expandable
             .expandable_header
-              .expandable_target.nav-secondary_link.nav-secondary_link__button
+              .expandable_target.nav-secondary_button
                 .expandable_header-left
                 .expandable_header-right
                   .expandable_cue-open
-                    .cf-icon.cf-icon-down.cf-icon__green
+                    .cf-icon.cf-icon-down
                   .expandable_cue-close
-                    .cf-icon.cf-icon-up.cf-icon__green
+                    .cf-icon.cf-icon-up
             .expandable_content
               .nav-secondary_list
                 .nav-secondary_item
-                  .nav-secondary_link / .nav-secondary_link.nav-secondary_link__current
+                  .nav-secondary_link /
+                  .nav-secondary_link.nav-secondary_link__current
   tags:
     - cfgov-nav-secondary
 */
 
 .nav-secondary {
+    .webfont-regular();
 
     .respond-to-max(@bp-sm-max, {
+        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
+
         background-color: @gray-10;
         border-top: 1px solid @gray-20;
         border-bottom: 1px solid @gray-20;
-        margin-bottom: unit(@grid_gutter-width / @base-font-size-px, em);
     });
 
-    // Inset horizontal rule between "IN THIS SECTION" text and page links.
-    & .inset-divider {
-      margin: 0 @grid_gutter-width / 2 @grid_gutter-width / 2;
-      border: 0;
-      height: 1px;
-      background-color: @gray-20;
+    &_button {
+        width: 100%;
+        padding: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
 
-      // Only shown on tablet/mobile sizes.
-      .respond-to-min(@bp-lg-min, {
-        display: none;
-      });
+        .webfont-demi();
+
+        text-align: left;
+        text-transform: uppercase;
+
+        &:focus {
+            outline: 1px dotted;
+        }
+
+        .cf-icon {
+            color: @green;
+        }
+
+        .respond-to-min(@bp-med-min, {
+            display: none;
+        });
     }
 
     &_list {
         margin: 0;
         padding: 0;
-    }
-    &_item, &_item__child {
-        list-style: none;
-        .webfont-regular();
 
-        .respond-to-min(@bp-lg-min, {
-            & + & {
-                margin-top: 10px;
-            }
-        });
-    }
+        &__parents {
+            .respond-to-max(@bp-sm-max, {
+                padding-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
+                margin-right: unit( (@grid_gutter-width / 2) / @base-font-size-px, em);
 
-    &_item {
-        .respond-to-min(@bp-med-min, {
-            & + &,
-            &__child {
-              margin-top: 10px;
-            }
-        });
-    }
+                border-top: 1px solid @gray-50;
```

Comment:
Should this use a variable like https://github.com/cfpb/cfgov-refresh/pull/1144/files#diff-b745345cfdbaf1327c1cb3a2a77a4d1eR10 ?